crates/integration_test/src/tests/lifecycle/container_lifecycle.rs

```diff
@@ -64,6 +69,7 @@ impl<'a> TestableGroup<'a> for ContainerLifecycle {
         vec![
             ("create", self.create()),
             ("start", self.start()),
+            // ("exec", self.exec("true")),
```

Comment:
Can we change the true command to something that gives output like `echo "Hello world"` ? That way we will be able to see if the exec command actually executes the given command inside the container. Otherwise even a no-op exec will pass the test.

Reply:
👍